packages/vm/scripts/benchmarks/mainnetBlocks.ts

```diff
@@ -1,76 +1,56 @@
 import * as fs from 'fs'
-import { merge } from 'lodash'
 import Account from '@ethereumjs/account'
 import { bufferToInt, toBuffer, setLengthLeft } from 'ethereumjs-util'
 import { encode } from 'rlp'
 import blockFromRPC from '@ethereumjs/block/dist/from-rpc'
 import VM from '../../dist'
 import { StateManager, DefaultStateManager } from '../../dist/state'
 import BN = require('bn.js')
+import Benchmark = require('benchmark')
+
+const suite = new Benchmark.Suite()
 
 async function main() {
   const args = process.argv
-  if (args.length < 3 || args.length > 4) {
+  if (args.length !== 3 && args.length !== 4) {
```

Comment:
Not a big deal but just wonder why you changed that evaluation? Code before was more generic and more flexibly allowed for range checks? 🤔

But again, in the end I really don't care, was just wondering. 😄

Reply:
oh i was just tinkering around with the script and found that logic easier to parse ;p i can change it back if you want!

Reply:
Don't worry, will mark as resolved. :-)